SU-HSDO/suhumsci
# Shs 53 mega menu v2 backend configuration

# READY FOR REVIEW

## Summary
Adds configuration for the mega menu inthe theme form.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Go to http://sparkbox-sandbox.suhumsci.loc/admin/appearance/settings/humsci_colorful
2. Select  Use megamenu on desktop
3. Go to the front page. The mega menu should be visible, and the regular menu hidden.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
